sw/device/tests/crypto/sha256_functest.c

```diff
@@ -182,6 +183,7 @@ OTTF_DEFINE_TEST_CONFIG();
 
 bool test_main(void) {
   status_t test_result = OK_STATUS();
+  CHECK_STATUS_OK(entropy_complex_init());
```

Comment:
Is it worth noting somewhere that not the hardware accelerator itself but the cryptolib or rather the function used to interface the hardware accelerator require entropy? Because the HMAC hardware IP really doesn't have an EDN interface.